tests/data/out_teacup.csv

```diff
@@ -0,0 +1,242 @@
+Time,Characteristic Time,Heat Loss to Room,Room Temperature,Teacup Temperature
```

Comment:
Do these `.csv` files want to be .gitignored? I'm assuming they are intermediate outputs from the test?

Reply:
No, they are used for the benchmarking tests. To check the output of assert_frames_close with different configurations and loading transposed files.